pygenomeworks/genomeworks/cudapoa/cudapoa.pyx

```diff
@@ -75,17 +75,15 @@ cdef class CudaPoaBatch:
             max_sequence_size,
             max_gpu_mem,
             output_type="consensus",
+            band_mode="static_band",
```

Comment:
In cpp, default value for `BatchConfig` constructor is `full_band`. That is to minimize the change in `cudapoatests`. But default value in `get_multi_batch_sizes` used `cudapoa-bin` and `sample_cudapoa` is `adaptive_band`. Should you change the default to full?

Reply:
I can change this to adaptive too if needed. I am not sure if defaulting it to full would make sense as it will not be able to run on a lot of lower-mem gpus.

Reply:
in cpp, we basically over-write the default in `cudapoa-bin` to be adaptive as well. So, yes, changing to adaptive should be correct. If you think it's better, I can change `BatchConfig` to have default of adaptive as well, and change the `cudapoatests` accordingly.

Reply:
I've changed it to adaptive as well. I think default full/adaptive is more like a policy decision for the app, but we should have it uniform throughout the codebase. If adaptive is the way to go for the cudapoa-bin, I think it makes sense to propagate that internally.